src/util/util.py

```diff
@@ -0,0 +1,23 @@
+def get_first_name(update):
+    return update.message.chat.first_name
+
+
+def parse_menu(template, data):
```

Comment:
Why don't we use the keys from `data` directly?

Reply:
I tried. Apparently data from psycopg2 with CursorDict is in the form of psycopg2.extras.DictRow, a dict-like data structure which can't play well in enhanced for loop. I'd have to make do with a template.

Reply:
What do you mean by an enhanced for loop? Can we do `for category in data.keys()`?

Reply:
data.keys() returns <odict_iterator object at 0x10dcd6e90>, whereas data.keys returns 
<bound method DictRow.keys of [datetime.date(2020, 1, 26), ['Salad Bar', 'Fruits', 'Cordial/Infused Water', 'Coffee/Tea/Milo'], ['Lamb Chop w/ Gravy', 'Nai Bai w/ Garlic', 'Cheesy Mashed Potato'], ['Dumpling Noodle'], ['White & Brown Rice', 'Chicken Thigh w/ Marinara Tomato Sauce', 'Crispy Fish w/ Nacho Cheese Sauce', 'Braised Cabbage w/ Black Fungus', 'Sweet & Sour Chicken Frank w/ Potato'], ['Sambal Mock Prawn', 'Sambal Veg Fish', 'Stir Fried Green Bean w/ Lotus Root & Carrot', 'Stir Fried You Mai Cai w/ Fermanted Beancurd'], ['Beef Goulash', 'Pasta Aglio Olio', 'Honey Carrots'], ['Black Dal', 'Longbean Poryal', 'Tandoori Chicken', 'Samosa x 2', 'Veg Fried Noodles', 'White Rice'], ['Cream Of Cauliflower Soup', 'Cabbage Soup']]>

Reply:
neither produce the key i could use

Reply:
Did you try iterating through the keys in `data.keys()?`

Reply:
I used `for key in data.keys()` and the error shows `'datetime.date' object is not iterable`
I suspect it is still looping an array